### DISCUSSION TITLE
AI Conferences Papers for ASReview

### DISCUSSION THREAD

**Discussion by george-gca** [asreview/asreview#1504] [Show and tell]
Since I have to read a lot of papers from top AI conferences, I created a file with the needed information from papers (title, abstract, url, authors) from these conferences for the last years. This way I could use ASReview to help discovering hidden gems that get lost in the sea of papers.

I believe this might be useful for more people, so I created a [GitHub repo](https://github.com/george-gca/ai_conferences_info/tree/main/). Currently there is data from: AAAI, ACL, COLING, CVPR, EACL, ECCV, EMNLP, Findings of the Association for Computational Linguistics, ICCV, ICLR, ICML, IJCAI, IJCNLP, KDD, NAACL, NeurIPS, SIGCHI, SIGDIAL, SIGGRAPH, TACL, and WACV.

Note that this data is not labeled (it doesn't have any paper flagged as useful/not useful), so it is a starting point for you to find papers that are good for your specific case.

**Comment by Rensvandeschoot** [comment]
Great work!!! Can you also add this information to this thread: https://github.com/asreview/asreview/discussions/1140

**Comment by J535D165** [comment]
I like this contribution. It would be nice to integrate this somehow in ASReview one day so people only have to click and start labeling directly. We will put this on the backlog and work on this for sure.

**Reply by george-gca** [reply]
Maybe it could have something like a [hugging face datasets](https://huggingface.co/docs/datasets/index) style, so that you could add your own dataset to a dataset registry that would be easily imported to your ASReview.
